recipes/gin-config/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gin-config" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6a83b7639ae76c276c0380d71d583f151b327a7c37978add314180ec1280a6cc
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - six >=1.10.0
+    - tensorflow
+
+test:
+  imports:
+    - gin
+    - gin.testdata
+    - gin.tf
```

Comment:
You should only check for `gin`, the others are only optional.
```suggestion
```

Reply:
@xhochy wouldn't i want to include `tensorflow` in the build in case there is required use for `gin.tf`?

Reply:
No, python dependencies aren't necessary in the build and add them to run makes them non-optional; and especially tensorflow is quite a heavy-weight dependency which we should keep optional if possible.

Reply:
If there are only certain versions of `tensorflow` with which `gin` works together, you could add a `run_constrained` for that.-

Reply:
@xhochy i went ahead and added a run constrained. should be good to go now. thanks!